src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -74,6 +75,21 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.SharedFramework.Sdk" />
 
   <Target Name="ResolveReadyToRunCompilers" DependsOnTargets="ResolveRuntimeFilesFromLocalBuild">
+    <!-- The following property group can be simplified once runtime repo switches over to SDK 6.0 drop -->
+    <PropertyGroup>
+      <CrossDir></CrossDir>
```

Comment:
nit:
```suggestion
      <CrossDir />
```

Reply:
Oh, sure, good point, thanks for spotting.

Reply:
Fixed in 2nd commit (in ReadyToRun.targets where I moved the logic that was originally duplicated between the two sfxproj scripts).